src/v/kafka/server/group_manager.cc

```diff
@@ -406,6 +406,13 @@ ss::future<> group_manager::recover_partition(
               "Recovering {} - {}",
               group_id,
               group_stm.get_metadata());
+            for (const auto& member : group_stm.get_metadata().members) {
```

Comment:
`group_stm.get_metadata().members` this is one of those cases where if `get_metadata` ever changes to return a value, then this loop will contain a UAF bug.

c++20 let's us add an init statement to the for loop to deal with it `for (auto x = value; auto& xx : x) { ... }`